docs/weightonlyquant.md

```diff
@@ -190,13 +190,14 @@ input_ids = tokenizer(prompt, return_tensors="pt").input_ids.to(device_map)
 config = WeightOnlyQuantConfig(weight_dtype="int4_fullrange",
                                algorithm="RTN",
                                group_size=32,
-                               compute_dtype="fp16",
-                               scale_dtype="fp16")
+                              )
 qmodel = AutoModelForCausalLM.from_pretrained(model_name, use_llm_runtime=False,
                                               device_map=device_map,quantization_config=config,
                                               trust_remote_code=True, torch_dtype=torchfloat16)
 
-# saving model, it should be executed before ipex.optimize_transformers function is called. 

```

Comment:
generate?
we should move save and load into below session or automatic hiding it